pages/nft-book-store/new.vue

```diff
@@ -316,6 +379,31 @@ async function onSubmit () {
   }
 }
 
+async function submitEditedClass () {
+  try {
+    isLoading.value = true
```

Comment:
Not sure if we need to guard if we are really in edit mode, and `editionIndex` is properly set here